spec/services/otp_sender_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe '#send_otp' do
     context 'when user is two_factor_enabled and does not have unconfirmed_mobile' do
       it 'sends OTP to mobile' do
-        user = build_stubbed(:user)
+        user = build(:user)
```

Comment:
You should be able to use `build_stubbed` here since the DB is not accessed in this scenario.